src/libNode/FallbackBlockProcessing.cpp

```diff
@@ -290,7 +290,24 @@ bool Node::ProcessFallbackBlock(const bytes& message, unsigned int cur_offset,
                                      *m_mediator.m_DSCommittee,
                                      m_mediator.m_ds->m_shards);
     }
-    StoreState();
+
+    auto writeStateToDisk = [this]() mutable -> void {
```

Comment:
writeStateToDisk has been defined 4 times, can be optimized?

Reply:
There are some slight difference among them, so I think better to keep it customized and keep on stack. In the later enhancement I may change this part totally.